src/main/resources/templates/application/opponents/opponents-remove.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>LAA CCMS - Start new application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a th:href="@{/application/summary/opponents}" class="govuk-back-link">Cancel and return to opponents and other parties</a>
+  </div>
+  <div id="main-content">
+    <form action="#" th:action="@{/application/opponents/{opponentId}/remove(opponentId=${opponentId})}" method="post">
+      <h1 class="govuk-heading-l">
+        Delete opponent
+      </h1>
+      <p class="govuk-body-l">
+        Are you sure you want to delete this opponent? Please click 'Confirm' to delete
```

Comment:
Do you need to specify the opponent we are deleting?

Reply:
opponent party name added to view